net/blockwise/blockwise.go

```diff
@@ -193,20 +159,18 @@ func newRequestGuard(request Message) *messageGuard {
 
 // NewBlockWise provides blockwise.
 // getSentRequestFromOutside must returns a copy of request which will be released by function releaseMessage after use.
```

Comment:
NewBlockWise -> New and is that bit about releaseMessage still valid?

Reply:
fixed